webapp/src/styles.scss

```diff
@@ -102,3 +111,6 @@ button {
 	}
 }
 
+input[type="file"] {
```

Comment:
i wouldn't add it to global styles. Someday someone wants to add another file input and gets confused as to why it's not there

Reply:
I changed it to make it a bit more obvious although we probably won't add visible file inputs as they are not supported by Angular Material